### ISSUE TITLE
Some of the progress bars are not visible in high contrast mode

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4643]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.4.3 Contrast (Minimum)

**Test Environment:**
- Test URL: https://clarity.design/documentation/progress
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0 and Edge Version 83.0.478.33 (Official build) beta (64-bit)

**Steps to reproduce:**
1. Enable high contrast mode on windows (left shift + left alt + Print Screen key combo).
2. Open the test URL.
3. Check the different progress bars on the page.

**Expected result:**
All the progress bars should be visible in the high contrast mode without losing the actual info. Where it's not feasible to achieve that, please provide alternatives to access that info.

**Actual result:**
Some of the progress bars are not visible in high contrast mode. The impacted progress bars falls under below categories:
1. Static progress bars.
2. Complex layout.
3. Stacked layout.

**Note:** ![Sampled reference screenshot](https://imgur.com/a/VPRF88E)
